consensus/src/liveness/round_state.rs

```diff
@@ -225,7 +228,7 @@ impl RoundState {
 
     /// Return if already voted for timeout
     pub fn is_vote_timeout(&self) -> bool {
```

Comment:
the name is a bit confusing, this should be `already_vote_timeout` or something?